services/housetables/src/main/java/com/linkedin/openhouse/housetables/repository/impl/jdbc/ToggleStatusHtsJdbcRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.linkedin.openhouse.housetables.repository.impl.jdbc;
+
+import com.linkedin.openhouse.housetables.model.TableToggleRule;
+import com.linkedin.openhouse.housetables.repository.HtsRepository;
+
+public interface ToggleStatusHtsJdbcRepository extends HtsRepository<TableToggleRule, Long> {
+  Iterable<TableToggleRule> findAllByFeature(String feature);
```

Comment:
addressed

Reply:
Can we also rename the method

Reply:
I thought about this. So to have it called by "feature" instead of feature Id is actually intentional. 
- At table level: I put `id` in table, db and feature indicating they are "as it is, only literally matching is allowed. 
- At HTS level: db/table were postfixed with `pattern` in the internal data model, as each row in mysql can be used for fuzzy matching. 

I don' know if we ever need to allow `feature` to accept regex. But didn't want to close that door.